### ISSUE TITLE
Searching for patient is not showing correctly always

### ISSUE THREAD

**Issue by archana900** [someshkar/covid19india-cluster#129]
**Describe the bug**
Searching for patient is not showing correctly always

**To Reproduce**
1. Search for P1 - P1 appears on screen
2. Search for P2 - P1 appears on screen
3. Search for P3 - P3 does not appears on screen (maybe because of the zoom levels due to previous searches of P1, and P2)

**Expected behavior**
P3 should have been visible in the frame
![Error](https://user-images.githubusercontent.com/26739031/77758503-1716ec80-7059-11ea-847a-7d1ff91b0e1d.png)

**Screenshots**

**Desktop (please complete the following information):**
 - Win 10
 - Chrome
 - Version 77.0.3865.120 (Official Build) (64-bit)

**Additional context**

**Comment by someshkar** [comment]
@SahajR can you have a look?

**Comment by SahajR** [comment]
Yeah sure. But before moving forward with this, I want to know if this is related to the non-centered zoom to the result patient. This is happening because the graph is under `simulation` and move outward gradually. One way to fix this is to stop the simulation, but then the clusters will not diverge from each other.

**Comment by SahajR** [comment]
Maybe pausing the simulation whilst searching, and then  resuming after a delay can help with this one. What do you say?
